Formula/jxrlib.rb

```diff
@@ -15,15 +15,14 @@ class Jxrlib < Formula
     sha256 cellar: :any_skip_relocation, yosemite:      "0dae977caf9e34289c9dd09f7e12bdf7158ccc42d9fa2dc00b4164b82c1caf3f"
   end
 
+  depends_on "cmake" => :build
+
+  patch :DATA
```

Comment:
Where did this patch come from? Has it been submitted upstream?

Reply:
Upstream has not received any commits in forever even the "official" fork on GitHub only has a marked release to appease package vcpkg.

I know debian, arch-aur and vcpkg do something similar, I'd tweaked the patch to more closely match what make does. This was originally discussed on https://jxrlib.codeplex.com/discussions/440294 by Mathieu Malaterre <malat@debian.org>, I'm also doing this in my own macports overlay due to it being a newer wine requirement.